.gitignore

```diff
@@ -488,3 +488,5 @@ _site/
 
 # Pytest profile files
 prof/
+*.orig
+.vscode/settings.json
```

Comment:
Create a new issue for changes you want to make that are unrelated to this pr.
Also, `.vscode` already exists in `.gitignore`.

Reply:
Created https://github.com/coala/community/issues/194